changelog.d/1068.change.rst

```diff
@@ -0,0 +1 @@
+filters ``exclude`` and ``include`` now support to pass literal name of attribtes in ``str`` format
```

Comment:
We are doing Markdown in changelogs now. :) (also please use the fully-qualified name instead of a description; i.e. "`attrs.filters.exclude()` and `include()` now..."

Reply:
Also with `{func}`?